src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComEventsMethod.cs

```diff
@@ -2,96 +2,139 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-/*============================================================
-**
-**
-** Purpose: part of ComEventHelpers APIs which allow binding 
-** managed delegates to COM's connection point based events.
-**
-**/
-
 using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Diagnostics;
+using System.Text;
 using System.Runtime.InteropServices;
 using System.Reflection;
 
-
 namespace System.Runtime.InteropServices
 {
-    // see code:ComEventsHelper#ComEventsArchitecture
+    /// <summary>
+    /// Part of ComEventHelpers APIs which allow binding
+    /// managed delegates to COM's connection point based events.
+    /// see code:ComEventsHelper#ComEventsArchitecture
```

Comment:
There's a few of these old code references to what I assume are framework docs. We should either remove these docs or add the doc to somewhere that the community can read them like docs or the BOTR.

Reply:
Those still exist. See `ComEventsHelper.cs`